Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/MainActivity.cs

```diff
@@ -6,15 +6,22 @@
 using Android.Content;
 using Android.Content.PM;
 using Android.OS;
+using Contoso.Forms.Demo.Droid;
 using Microsoft.AppCenter.Push;
+using Xamarin.Forms;
 
+[assembly: Dependency(typeof(MainActivity))]
```

Comment:
what is that?

Reply:
Each implementation of the interface needs to be registered with a `DependencyService` that has a metadata property. After the implementation is complete, we also use the `[assembly]` tag to register them. Only then can the `DependencyService` find the implementation class corresponding to the interface at runtime.

Reply:
Ok thanks